src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1330,7 +1330,7 @@ public async Task<List<FieldTypeClass>> GetTypeFields(int typeId, CancellationTo
                 //.Replace("System.Decimal", "decimal")
                 .ToString();
 
-        internal async Task<MonoBinaryReader> GetCAttrsFromType(int objectId, int typeId, string attrName, CancellationToken token)
+        internal async Task<MonoBinaryReader> GetCAttrsFromType(int typeId, string attrName, CancellationToken token)
```

Comment:
Why the change? Needs tests that would fail with `objectId`, and now pass with `typeId`.

Reply:
Argument `objectId` was never used but passes nevertheless. It was confusing so I removed it. `TypeId` was there all the time